### ISSUE TITLE
Port To Apple TV (tvOS)

### ISSUE THREAD

**Issue by fkick** [inickt/LazyMan-iOS#57]
Is it possible to port this to tvOS? With Plex plugin support gone, this would be great to get on the Apple TV and Apple TV 4K.

**Comment by inickt** [comment]
Yeah I am working on it. I did some refactoring over the summer to make it easier to port but have run out of free time recently. Going to try and finish Chromecast support and then get an Apple TV app going.

**Comment by fkick** [comment]
WIth Lazyman now back, any chance of the AppleTV port?

[https://www.reddit.com/r/LazyMan/comments/lzzve5/welcome_back/](url)

**Comment by inickt** [comment]
Knowing that it is back and supported I may do a basic rewrite in SwiftUI so I can knock out iOS/tvOS/macOS all at once. I can't commit to anything but it makes me feel a bit better about maintaining it.

**Comment by Galorlx** [comment]
You got anything connecting back to tvOS does the IPA work for all or what’s the stages on right now?
